src/group_theory/coset.lean

```diff
@@ -376,8 +376,41 @@ calc α ≃ Σ L : quotient s, {x : α // (x : quotient s) = L} :
     ... ≃ quotient s × s :
   equiv.sigma_equiv_prod _ _
 
-lemma card_eq_card_quotient_mul_card_subgroup [fintype α] (s : subgroup α) [fintype s]
-  [decidable_pred (λ a, a ∈ s)] : fintype.card α = fintype.card (quotient s) * fintype.card s :=
+variables {t : subgroup α}
+
+/-- If `H ≤ K` then `G/H ≃ G/K × K/H`. -/
```

Comment:
Could you extend the doc-string to say explicitly that this is the constructive version, which requires the user to provide a section of the quotient map, and provide a pointer to `quotient_equiv_prod_of_le` for the classical version?